ceph-releases/reef/centos-arm64/__GANESHA_PACKAGES__

```diff
@@ -1 +1 @@
-../../../src/daemon-base/__GANESHA_PACKAGES__
\ No newline at end of file
+../centos/__GANESHA_PACKAGES__
```

Comment:
@guits this change makes it so the arm builds also pin to the same versions as the x86 ones. https://tracker.ceph.com/issues/63151